xilem/examples/to_do_mvc.rs

```diff
@@ -6,9 +6,13 @@
 // On Windows platform, don't show a console when opening the app.
 #![windows_subsystem = "windows"]
 
+use masonry::properties::types::Length;
+use masonry::theme::{DEFAULT_GAP, ZYNC_800};
 use winit::error::EventLoopError;
 use xilem::style::Style as _;
-use xilem::view::{button, checkbox, flex_col, flex_row, label, text_input};
+use xilem::view::{
+    FlexExt, FlexSpacer, button, checkbox, flex_col, flex_row, label, text_button, text_input,
+};
 use xilem::{EventLoop, EventLoopBuilder, InsertNewline, WidgetView, WindowOptions, Xilem};
 use xilem_core::Edit;
```

Comment:
While I see this (still fits the title, same issue as above).
```suggestion
use xilem::core::Edit;
```